src/Resources/skeleton/registration/twig_email.tpl.php

```diff
@@ -3,7 +3,7 @@
 <p>
     Please confirm your email address by clicking the following link: <br><br>
     <a href="{{ signedUrl|raw }}">Confirm my Email</a>.
-    This link will expire in {{ expiresAt|date('g') }} hour(s).
+    This link will expire in {{ expiresAt|date('g', 'UTC') }} hour(s).
```

Comment:
I'm probably missing something, but the `g` format outputs "12-hour format of an hour without leading zeros". So, if it's 8:35 AM it outputs `8` and if it's  17:21 PM it outputs `5`.

that's why I think the text requires some rewording like this:

```
will expire in ... hour(s) --> will expire at ...
```